kodilla-rps/src/main/java/Round.java

```diff
@@ -0,0 +1,46 @@
+public class Round {
+
+    ActionType playerAction;
+    ActionType computerAction;
+    private Player player;
+    private Computer computer;
+
+    public Round(Player player, Computer computer) {
```

Comment:
here you make an assumption that there is one human player, and another one is computer. How to make it more flexible and allow games between 2 human players?

Reply:
done